middleware/requestid/requestid.go

```diff
@@ -31,9 +31,9 @@ type Config struct {
 
 // ConfigDefault is the default config
 var ConfigDefault = Config{
-	Next:      nil,
-	Header:    fiber.HeaderXRequestID,
-	Generator: utils.UUID,
+	Next:       nil,
```

Comment:
Please remove the requestid mw related stuff from this PR. You can make another one of course:smile: